bin/check-port.rb

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env ruby
+#
+#  encoding: UTF-8
+#   check-port
```

Comment:
why is check-port being added here? It's a duplicate of check-ports from sensu-plugins-network-checks.